README.md

```diff
@@ -79,20 +80,31 @@ Install this project with:
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ### Usage
+
 > To run the project, execute the following command:
+
 ```sh
    rails server
 ```
 
 ## 👥 Authors <a name="authors"></a>
 
-
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍